app/assets/stylesheets/application.scss

```diff
@@ -30,13 +44,37 @@ footer {
   margin-top: 0;
 }
 
+.brand-logo {
+  margin-left: 10px;
+}
+
+.hero-unit {
+  margin-top: 1vh;
```

Comment:
Properties should be ordered background-image, background-size, height, margin-top, z-index